src/pages/faq.hbs

```diff
@@ -0,0 +1,47 @@
+{{#extend "raha/layout/base/layout"}}
+  {{#content "styles" mode="append"}}
+    .faq {
+      margin-top: 30px;
+      margin-bottom: 30px;
+    }
+
+    .anchorOffset {
+      display: block;
+      position: relative;
+      top: -80px;
+      visibility: hidden;
+    }
+    
+    .bareLink {
+      text-decoration: none;
+    }
+  {{/content}}
+
+  {{#content "body"}}
+    <main class="faq">
+      <article>
+        <h1 class="anchorOffset" id="top" data-t>faqTitle</h1>
+        
+        {{#each faq as |sectionValue sectionKey|}}
+            <h3>{{sectionKey}}</h3>
```

Comment:
Downside of this is that it's not internationalizable. It's fine to just have a comment here expressing that. Better solution is to just have a `sectionTitle` field for each section key.